test/framework/package.py

```diff
@@ -184,10 +184,16 @@ def test_active_pns(self):
 
         pns = ActivePNS()
 
-        # default: EasyBuild package naming scheme, pkg release 1
-        self.assertEqual(pns.name(ec), 'OpenMPI-2.1.2-GCC-6.4.0-2.28')
-        self.assertEqual(pns.version(ec), 'eb-%s' % EASYBUILD_VERSION)
-        self.assertEqual(pns.release(ec), '1')
+        if get_package_naming_scheme() == "EasyBuildPNS":
+            # default: EasyBuild package naming scheme, pkg release 1
+            self.assertEqual(pns.name(ec), 'OpenMPI-2.1.2-GCC-6.4.0-2.28')
+            self.assertEqual(pns.version(ec), 'eb-%s' % EASYBUILD_VERSION)
+            self.assertEqual(pns.release(ec), '1')
+        elif get_package_naming_scheme() == "EasyBuildDebFriendlyPNS":
+            # default: EasyBuild deb friendly package naming scheme, pkg release 1
+            self.assertEqual(pns.name(ec), 'OpenMPI-2.1.2-GCC-6.4.0-2.28')
+            self.assertEqual(pns.version(ec), '%s_eb' % EASYBUILD_VERSION)
```

Comment:
Shoulding this be
```suggestion
            self.assertEqual(pns.version(ec), '%s-eb' % EASYBUILD_VERSION)
```
(or the return value from `pns.version(ec)` changed)?